FoursquareTopTests/VenueViewModel+Builder.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  VenueViewModel+Builder.swift
+//  FoursquareTop
+//
+//  Created by Fran on 9/21/16.
+//  Copyright © 2016 Karumi. All rights reserved.
+
+
+import Foundation
+import MapKit
+
+@testable import FoursquareTop
+
+extension VenueViewModel {
```

Comment:
I know that @gragera use Builder in this app, but it could have been done, also by adding init() method by extension. Seems more Swifty.